react-client/src/Components/App.js

```diff
@@ -42,59 +41,79 @@ function App() {
     return () => {};
   }, []);
 
-  useEffect(() => {
-    return () => {};
-  }, []);
-
   const injectDOMEventInBody = async () => {
     document.addEventListener("click", (e) => {
       if (e.target.id !== "closedompeeker" && domSwitch == true) {
         e.preventDefault();
 
-        const children = [...e.target.children].map(child => {
-          return { id: child.id, class: child.className };
+        const children = [...e.target.children].map((child) => {
+          return {
+            id: child.id.trim() !== "" ? "#" + child.id : null,  class: child.className.trim() !== "" ? "." + child.className : null, childtag: child.localName
+          };
         });
 
+        if (e.target.id.trim() !== "") {
+          var eltarget = document.querySelector(`#${e.target.id}`);
```

Comment:
Is `eltarget` DOM query even necessary? Why bother querying the element when the element itself is already called in `e.target`?

Reply:
Not only that, putting variable inside a conditional statement when you'll use it outside of that scope is a bad practice. Don't do this.

Reply:
done didnot  expect na puede mag run using e.target lng but  already fixed and if functions are now removed